tests/amqpexchange_declare_existent.phpt

```diff
@@ -23,6 +23,7 @@ try {
     $ex->setName($exchangge_name);
     $ex->setType(AMQP_EX_TYPE_TOPIC);
     $ex->declareExchange();
+    echo 'exchange ', $ex->getName(), ' declared', PHP_EOL;
```

Comment:
Not sure I get the change, shouldn’t there be a related change in the EXPECT section?